nimble/controller/src/ble_ll_ctrl.c

```diff
@@ -2540,6 +2542,17 @@ ble_ll_ctrl_tx_done(struct os_mbuf *txpdu, struct ble_ll_conn_sm *connsm)
             connsm->enc_data.enc_state = CONN_ENC_S_PAUSE_ENC_RSP_WAIT;
         }
         break;
+    case BLE_LL_CTRL_PHY_REQ:
+        if (connsm->conn_role == BLE_LL_CONN_ROLE_SLAVE) {
+            if (connsm->phy_data.req_pref_tx_phys_mask & BLE_PHY_MASK_1M) {
+                connsm->phy_tx_mod_transition = BLE_PHY_1M;
```

Comment:
`phy_tx_mod_transition` is not defined anywhere in this patch.
also what `mod` means? if this is typo in `mode` then you should not use `BLE_PHY_1M` here because this is not a mode